src/contracts/interfaces/IDelegationManager.sol

```diff
@@ -134,11 +134,12 @@ interface IDelegationManagerTypes {
         uint32 startTimestamp;
         // Array of strategies that the Withdrawal contains
         IStrategy[] strategies;
-        // TODO: Find a better name for this
-        // Array containing the amount of staker's scaledSharesToWithdraw for withdrawal in each Strategy in the `strategies` array
-        // Note that these shares need to be multiplied by the operator's totalMagnitude at completion to include
-        // slashing occurring during the queue withdrawal delay
-        uint256[] scaledSharesToWithdraw;
+        // Array containing the amount of staker's scaledShares for withdrawal in each Strategy in the `strategies` array
+        // Note that these scaledShares need to be multiplied by the operator's maxMagnitude at completion to include
```

Comment:
Shouldn't we also add beacon chain scaling factor here?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/842/commits/ba893331d545b5feb09573287e5f37b1fc63a27c#diff-d3c92783ec657b412dca73e18454a61feff67efa9b92f6e72a48eaa863a2626a
Added note on that